app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,12 @@
+class MechanicsController < ApplicationController
+  def show
+    @mechanic = Mechanic.find(params[:id])
+  end
+
+  def update
```

Comment:
Per your reflections, this sorta works but is not entirely RESTful. I will look at your form and see what you have. 
To be RESTful, we would want to create a `MechanicRidesContorller` with a `create` action. The form method would then need to be a `post`.